demo/src/index.html

```diff
@@ -12,6 +12,7 @@
       integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
       crossorigin="anonymous"
     />
+    <link rel="icon" href="data:," />
```

Comment:
Hmm, out of curiosity what does this line do exactly to fix it?

Reply:
I don't see the error, but what if we just completely remove the favicon line?  We don't need it in consuming applications, and there's really no need in the standalone app.

Reply:
The browser always looks for favicon even when we don't specify it within the HTML. I couldn't find any line for requesting a favicon in the code. This line responds to the favicon request by the browser.

Reply:
Ok, yea that line above, 15, is what requests the `favicon`.   So yea either way...leave as is or remove line #15 and think we're good on this.

Reply:
Other solutions I found are;
1. Stop the browser the from making the favicon request
2. Respond to the browser request for favicon with an empty line (this solution)
3. Catch the favicon request and send a HTTP 204 status within the app
4. Provide a favicon file (which we want to avoid)